Doc/library/sqlite3.rst

```diff
@@ -47,85 +47,177 @@ This document includes four main sections:
       PEP written by Marc-André Lemburg.
 
 
+.. We use the following practises for SQL code:
+   - UPPERCASE for keywords
+   - snake_case for schema
+   - single quotes for string literals
+   - singular for table names
+   - if needed, use double quotes for table and column names
+
 .. _sqlite3-tutorial:
 
 Tutorial
 --------
 
-To use the module, start by creating a :class:`Connection` object that
-represents the database.  Here the data will be stored in the
-:file:`example.db` file::
+In this tutorial, you will create a database of Monty Python movies,
+using basic :mod:`!sqlite3` APIs.
```

Comment:
```suggestion
In this tutorial, you will create a database of Monty Python movies
using basic :mod:`!sqlite3` functionality.
```

* "APIs" -> "functionality", as its more concrete, clear and not every beginner will necessarily be familiar with it
* Elide unnecessary comma